tests/testthat/test_extension_path.R

```diff
@@ -0,0 +1,8 @@
+skip_on_cran()
+skip_on_os(c("windows"))
+
+test_that("duckdb extensions are in a R specific directory", {
+  con <- DBI::dbConnect(duckdb(config=list('allow_unsigned_extensions'='true')))
+  on.exit(DBI::dbDisconnect(con, shutdown = TRUE))
+  expect_error(dbExecute(con, "LOAD 'httpfs'"),regexp = ".*R/duckdb/extensions/.*/httpfs.duckdb_extension\" not found.")
```

Comment:
Can this test be possibly done on a non-existing extension? Something like `LOAD 'not_existing_extension'` and checking for `duckdb/extensions/.*/not_existing_exension.duckdb_extension` in the error message?
Unsure in what conditions this test is run, but I think this will fail if httpfs happens to be there.